packages/web3-eth/src/web3_subscriptions.ts

```diff
@@ -96,6 +144,11 @@ export class NewHeadsSubscription extends Web3Subscription<
 	}
 }
 
+/**
+ * ## subscribe(“syncing”)
+ *
+ * Subscribe to syncing events. This will return an object when the node is syncing and when it’s finished syncing will return `false`
+ */
```

Comment:
it has changed event = true/false
and data event
example here packages/web3/test/integration/sync.test.ts